feedthefox/users/views.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import redirect, render
+
+from feedthefox.users import forms
+
+
+@login_required
+def view_profile(request):
+    user = request.user
+    newsletter_form = forms.ReceiveNewsLetterForm(request.POST or None,
+                                                  instance=user)
+
+    if newsletter_form.is_valid():
+        newsletter_form.save()
+        return redirect('view_profile')
+
+    data = {'user': user,
```

Comment:
We already have ``django.contrib.auth.context_processors.auth`` in ``context_processors`` so ``request.user`` is already in the context.